.github/workflows/flow-micro-benchmarks.yml

```
@@ -37,22 +37,22 @@ jobs:
             echo "baseline_exists=true" >> $GITHUB_OUTPUT
           else
             echo "baseline_exists=false" >> $GITHUB_OUTPUT
-          fi    
+          fi
       - name: Run benchmark on PR with baseline from master
         if: github.event_name == 'pull_request' && steps.check_baseline.outputs.baseline_exists == 'true'
-        run: cargo bench -- --baseline master
-        working-directory: src/redisearch_rs/trie_bencher
+        run: cargo bench --workspace -- --baseline master
+        working-directory: src/redisearch_rs
       - name: Run benchmark on PR without baseline
         if: github.event_name == 'pull_request' && steps.check_baseline.outputs.baseline_exists == 'false'
-        run: cargo bench
-        working-directory: src/redisearch_rs/trie_bencher
+        run: cargo bench --workspace
+        working-directory: src/redisearch_rs
       - name: Run benchmark on master
-        if: github.ref == 'refs/heads/master' && github.event_name == 'push' && success() 
-        run: cargo bench -- --save-baseline master
-        working-directory: src/redisearch_rs/trie_bencher
+        if: github.ref == 'refs/heads/master' && github.event_name == 'push' && success()
+        run: cargo bench --workspace -- --save-baseline master
+        working-directory: src/redisearch_rs
 
       - name: Upload rust baseline benchmarks for master
-        if: github.ref == 'refs/heads/master' && github.event_name == 'push' && success() 
+        if: github.ref == 'refs/heads/master' && github.event_name == 'push' && success()
```

```suggestion
        if: github.ref == 'refs/heads/master' && github.event_name == 'push'
```
That was a mistake - getting the baseline in the first step may fail.